src/v2/css/beam.css

```diff
@@ -1,563 +0,0 @@
-footer,header,hgroup,nav,section{display:block}
```

Comment:
If it's possible to leave this v2 directory around, I'd do so just to avoid putting more in this PR. But if it helps clean up the css because there are cross references or something, then go ahead.